target/riscv/insn_trans/trans_rvi.inc.c

```diff
@@ -278,6 +278,43 @@ static void gen_sltu(TCGv ret, TCGv s1, TCGv s2)
 
 static bool trans_slti(DisasContext *ctx, arg_slti *a)
 {
+#ifdef CONFIG_TCG_LOG_INSTR
+    /*
+     * If instruction tracing is enabled, we use addi zero, zero, <magic>
```

Comment:
Slti?

Reply:
I originally used addi, however @jrtc27 correctly pointed out that I was using a reserved opcode for HINT and suggested this alternative that is not compressible and not reserved.

Reply:
Yeah I just meant the comment is stale
```suggestion
     * If instruction tracing is enabled, we use slti zero, zero, <magic>
```